extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -32,6 +32,7 @@ attr_lists: { # amp-audio attributes that are common to both AMP and A4A format.
   attrs: { name: "controls" }
   attrs: { name: "loop"  value: "" }
   attrs: { name: "muted" value: "" }
+  attrs: { name: "controlsList" }
```

Comment:
Minor nit: it'd be good to keep the attributes alphabetize.

And if the only acceptable value for `controlsList` is `nodownload` then it can explicitly set as that:
```
attrs: {
  name: "controlsList"
  value: "nodownload"
}
```